junifer/markers/tests/test_brainprint.py

```diff
@@ -0,0 +1,47 @@
+"""Provide tests for BrainPrint."""
+
+# Authors: Synchon Mandal <s.mandal@fz-juelich.de>
+# License: AGPL
+
+import socket
+
+import pytest
+
+from junifer.datagrabber import DataladAOMICID1000
+from junifer.datareader import DefaultDataReader
+from junifer.markers import BrainPrint
+from junifer.pipeline.utils import _check_freesurfer
+
+
+def test_get_output_type() -> None:
+    """Test BrainPrint get_output_type()."""
+    marker = BrainPrint()
+    assert marker.get_output_type("FreeSurfer") == "vector"
+
+
+def test_validate() -> None:
+    """Test BrainPrint validate()."""
+    marker = BrainPrint()
+    assert set(marker.validate(["FreeSurfer"])) == {"scalar_table", "vector"}
+
+
+@pytest.mark.skipif(
+    _check_freesurfer() is False, reason="requires FreeSurfer to be in PATH"
+)
+@pytest.mark.skipif(
+    socket.gethostname() != "juseless",
```

Comment:
Why only for juseless?

Reply:
The data takes quite some time to fetch and the computation is better half of an hour, so would exceed the limitations of the gh-actions standard runners. We can pull it off on gh-actions if we have a small testing dataset providing FreeSurfer derivatives.